docs/intro.rst

```diff
@@ -73,31 +73,44 @@ needed and organize them in a local :doc:`datastore <dev/datastore>`.
 .. _etl-process:
 
 ---------------------------------------------------------------------------------------
-The ETL Process
+The Data Warehouse Design
 ---------------------------------------------------------------------------------------
 
-The core of PUDL's work takes place in the ETL (Extract, Transform, and Load)
-process.
+PUDL's data processing produces a data warehouse that can be used for analytics.
+The processing happens within Dagster assets that are persisted to storage,
+typically pickle, parquet or SQLite files. The raw data moves through three
+layers of the data warehouse.
```

Comment:
I wonder if the introduction page is the right place for each of the following layer depictions...maybe we should re-think what the introduction to PUDL is! Our docs are starting to get very extensive, and I want it to be obvious where people need to click for certain types of information and not have too much duplication.